superset/assets/javascripts/explorev2/actions/exploreActions.js

```diff
@@ -49,6 +53,33 @@ export function fetchFieldOptions(datasourceId, datasourceType) {
   };
 }
 
+export const FETCH_FAVE_STAR = 'FETCH_FAVE_STAR'
+export function fetchFaveStar(sliceId) {
+  return function (dispatch) {
+    const url = `${FAVESTAR_BASE_URL}/${sliceId}/count`;
```

Comment:
there is no such action is count, I believe you can keep it with no action and the endpoint will return you the count